README.md

```diff
@@ -1,5 +1,6 @@
 [<img width="300" src="https://static.getmonero.org/images/kovri/logo.png" alt="ˈKoʊvriː" />](https://github.com/monero-project/kovri)
 
+# Kovri
```

Comment:
Why do you want to add this line? It's redundant because of the logo (and adding this looks strange to me).